tests/logical_operators.rs

```diff
@@ -81,3 +81,25 @@ fn nesting() {
   evaluate("'' || '' || '' || '' || 'foo'", "foo");
   evaluate("'foo' && 'foo' && 'foo' && 'foo' && 'bar'", "bar");
 }
+
+#[test]
+fn empty_variadics_are_falsey() {
+  #[track_caller]
+  fn eval_variadics<'a>(args: impl AsRef<[&'a str]>, expected: &'a str) {
+    Test::new()
+      .justfile("@foo *args:\n echo {{ args && 'true' || 'false' }}")
```

Comment:
Maybe we should have separate tests for `&&` and `||`. Operator precedence is always confusing, so separating them might help comprehension.